client/src/pages/Profile.jsx

```diff
@@ -0,0 +1,20 @@
+import { Container  } from "react-bootstrap";
+import { Link } from 'react-router-dom'
+
+export const Profile = () => {
+  return (
+    <Container>
+      <div >
+        <br />
+        <br />
+        <br />
+        <br />
+        <br />
+      </div>

```

Comment:
I think we can replace this with css styling

Reply:
i copied this off jessicas code LMFAO
 but yes prolly could be done via css